chroma-bundles/include/find_deps.py

```diff
@@ -138,10 +138,10 @@ def get_deps(self, search_repos, pkg_list, excludes):
 yumdepfinder = YumDepFinder()
 deps, missing_deps = yumdepfinder.get_deps(repos, pkgs, excludes)
 
-if missing_deps:
-    print >>sys.stderr, "final missing_deps: %s" % missing_deps
-    print '\n'.join(missing_deps)
-    sys.exit(1)
+# if missing_deps:
```

Comment:
Any reason we can't just remove this?

Reply:
it needs to be enabled again when we switch back

Reply:
@johnsonw Indeed.  This whole bit of checking just needs to be smarter.  Right now it's hardcoded for the MFL production repo.  It needs to by dynamic to whichever repo is being used by the code.